### PR TITLE
Ops 1505

### PR DESCRIPTION
**Pull request by ztschafer**
Please review. This creates a new ToS but it is not in the nav and does not yet replace the existing one. It can only be navigated to via direct URL (policy.ovation.io/site/policies/tos-202208/index.html). The changes are as in the .docx linked to the JIRA ticket.

### CODE REVIEW

File: docs/policies/tos-202208.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# Terms of Service
```
Comment (michaeljon) [inline docs/policies/tos-202208.md:1]: Did we never have a ToS before? Is this a copy of the old file with a new name?

**Review by michaeljon — APPROVED** [review]
LGTM

**Comment by michaeljon** [comment]
Do we have a policy governing who can publish a new policy?

File: docs/policies/tos-202208.md (hunk 1 above)
Comment (ztschafer) [reply docs/policies/tos-202208.md:1]: We do have a ToS page. This is a copy of that w/ mods accessed under a different URL that is not accessible from the left-nav.

**Comment by ztschafer** [comment]
> Do we have a policy governing who can publish a new policy?

Turns out we do! But it's only that I have to approve. There should be a separation of duties, which we happen to be doing here, but it's not part of policy.

> Customer Contractual Agreements
> 
> Standard terms of service and associated BAA compliant with 45 CFR §164.308(b) must be approved by the Security Officer. These, or alternatives that are substantially similar, must be incorporated into customer contractual agreements. Alternatives must be approved by the Security Officer.

**Comment by michaeljon** [comment]
I'm overriding Zane's unsigned commits. We'll work out what happened there.

**Pull request merged**